woltka/cli.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020--, Qiyun Zhu.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Standalone command-line interface (CLI) of the program.
+"""
+
+import click
+
+from . import __version__
+from .workflow import workflow
+
+
+class NatOrder(click.Group):
```

Comment:
This might be small, but I see no reason to not be more explicit.

```suggestion
class NaturalOrderGroup(click.Group):
```

Reply:
Good point!